voice/src/main/java/discord4j/voice/DefaultVoiceGatewayClient.java

```diff
@@ -238,11 +240,13 @@ private Mono<Void> connect(VoiceServerOptions vso, String sessionId,
                                     .flatMap(payloadReader)
                                     .doOnNext(payload -> {
                                         if (payload instanceof Hello) {
+                                            log.debug("Received HELLO");
```

Comment:
i dont feel this and the READY debug are necesary or just can me moved from the start of if with a trace for print the class if necesary.